### DISCUSSION TITLE
Any way to quickly check the history lock of one place?

### DISCUSSION THREAD

**Discussion by atline** [labgrid-project/labgrid#979] [Q&A] [answered]
If one guy lock the place, and already unlocked it when I check issue. If possible I could use any command to find who ever locked the place?

**Comment by Bastian-Krause** [comment, marked as answer]
I am not sure I understand the question. Could you rephrase it?

Regarding lock history of a place, there is currently nothing except the coordinator log. See also #626.

Edit: Since #983 the coordinator logs acquire/lock and release/unlock operations.

**Reply by atline** [reply]
@Bastian-Krause 

The aims is same as #626, know who at which time lock the boards, so we can find that guy. Just I'm ok to find these information from coordinator.

But the coordinator log is really big (You mean the log printed by `crossbar start` if I understand correctly?), so could you point to me what kinds of keyword I need to search from that huge log?

**Reply by Bastian-Krause** [reply]
Yes, I meant the log printed by `crossbar start` and it's pretty big. It looks like I was wrong, acquiring (locking) and releasing (unlocking) places is not really visible there. For some reason I thought we would log the same information as `labgrid-client monitor` shows:
```
Place myplace changed:
  acquired: None -> myhost/bst
[...]
Place myplace changed:
  acquired: myhost/bst -> None
[...]
```
... but apparently we do not log this.

So the easiest fix would probably be printing a similar message in `CoordinatorComponent.acquire_place()` right before it returns `True` at the very end:

https://github.com/labgrid-project/labgrid/blob/414c88d272693279c86e839c54bc02d0d2593bba/labgrid/remote/coordinator.py#L624-L657

Would you be willing to create a PR for this?

**Reply by atline** [reply]
@Bastian-Krause Sure, here is it: https://github.com/labgrid-project/labgrid/pull/983

**Reply by Bastian-Krause** [reply]
#983 was merged.
